cmd/containers-storage/layers.go

```diff
@@ -9,10 +9,21 @@ import (
 	"github.com/containers/storage/pkg/mflag"
 )
 
-var listLayersTree = false
+var (
+	listLayersTree = false
+	cleanup        = false
+)
 
 func layers(flags *mflag.FlagSet, action string, m storage.Store, args []string) int {
-	layers, err := m.Layers()
+	var (
+		layers []storage.Layer
+		err    error
+	)
+	if cleanup {
```

Comment:
Why is this called cleanup?  It looks like you are only getting RWLayers and printing them?  Not sure what cleanup means in this case?

Reply:
I think we can remove the `cleanup` flag and stick to the `RWLayers`-only approach.